content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
+
+Istio captures a trace for all requests by default. Every time you
+access `/productpage` you see a corresponding trace in the Jaeger
+dashboard. This sampling rate is suitable for a test or low traffic
+mesh. For a high traffic mesh you can lower the trace sampling
+percentage in one of two ways:
+
+* During the mesh setup, use the Helm option `pilot.traceSampling` to
+  set the percentage of trace sampling. See the
+  [Helm Install](/docs/setup/kubernetes/helm-install/) documentation for
+  details on setting options.
+* In a running mesh, edit the `istio-pilot` deployment and
+  change the environment variable with the following steps:
+
+  1. To open your text editor with the deployment configuration file
+     loaded, run the following command:
+
+     {{< text bash >}}
```

Comment:
This needs to be indented a multiple of 4, and the content should be lined up with the {{